src/tests/test_package_loading.py

```diff
@@ -159,8 +159,8 @@ def test_load_failure_retry(selenium_standalone):
 
 
 def test_load_package_unknown(selenium_standalone):
-    url = selenium_standalone.server_hostname
-    port = selenium_standalone.server_port
+    selenium_standalone.server_hostname
+    selenium_standalone.server_port
```

Comment:
Not sure what these were for

Reply:
`load_package` didn't used to support relative urls, so below the url had to be something like `f"{url}:{port}/pyparsing-custom-3.0.6-py3-none-any.whl"`. At some point I changed it to use a relative URL since that is easier too update.